app/filtering.js

```diff
@@ -118,6 +118,11 @@ if (process.env.NODE_ENV === 'development') {
 function registerForBeforeRequest (session, partition) {
   const isPrivate = module.exports.isPrivate(partition)
   session.webRequest.onBeforeRequest((details, muonCb) => {
+    if (details.url && details.url.includes('chrome://brave/') &&
+      details.tabId !== -1) {
```

Comment:
tabs can be -1 here for renderer initiated new tabs/windows that start to load before they are assigned to a window

Reply:
done now